electron/electron.ts

```diff
@@ -53,7 +53,6 @@ async function createWindow() {
       devTools: isDev || IS_TEST,
       nodeIntegration: true,
       contextIsolation: false,
-      nativeWindowOpen: true,
```

Comment:
Now it's on by default: https://www.electronjs.org/docs/latest/breaking-changes#removed-nativewindowopen